vsm

```diff
@@ -103,6 +104,10 @@ class State(object):
     def got_signal(self, signal, value):
         vars(self.variables)[signal] = value
 
+        od = collections.OrderedDict(sorted(vars(self.variables).items()))
```

Comment:
You don't need to create an OrderedDict just for this, you can have sorted(vars(…)) in the for below…